wcivf/apps/parties/templates/parties/party_detail.html

```diff
@@ -107,6 +107,17 @@ <h3>{% trans "Ballot descriptions" %}</h3>
                 {% endfor %}
             </ul>
         {% endif %}
+
+        {% if object.emblems.all %}
+            <h3>{% trans "Emblems" %}</h3>
+            <div class="ds-grid">
```

Comment:
Two problems here:

1. I think the design system docs are wrong, and `ds-grid` needs to be set on the direct parent of the grid items (in this case, the `ul`.
2. The `ul` needs to have `li` and children, not `img`.

In addition to this, can you set `--gridCellMin: 5em;` on the `ul` (as per the docs)? This is to prevent the emblems getting too large on smaller screens (or on any screen, really)

Reply:
Updated, apologies for my Friday brain not using `li`s 🤦 

I've updated the screenshot too!